foundation/foundation/src/main/res/values/styles-texts.xml

```diff
@@ -2,143 +2,146 @@
         <item name="android:textColor">?attr/vtmnContentPrimary</item>
     </style>
 
     <style name="TextAppearance.Vitamin.Headline6" parent="@style/TextAppearance.MaterialComponents.Headline6">
-        <item name="fontFamily">@font/roboto_regular</item>
-        <item name="android:fontFamily">@font/roboto_regular</item>
+        <item name="fontFamily">@font/roboto_family</item>
+        <item name="android:fontFamily">@font/roboto_family</item>
         <item name="android:textSize">@dimen/vtmn_text_size_headline_6</item>
         <item name="lineHeight">@dimen/vtmn_line_height_headline_6</item>
         <item name="android:textStyle">bold</item>
-        <item name="fontWeight">700</item>
         <item name="android:textColor">?attr/vtmnContentPrimary</item>
     </style>
 
     <style name="TextAppearance.Vitamin.Subtitle1" parent="@style/TextAppearance.MaterialComponents.Subtitle1">
-        <item name="fontFamily">@font/roboto_regular</item>
-        <item name="android:fontFamily">@font/roboto_regular</item>
+        <item name="fontFamily">@font/roboto_family</item>
+        <item name="android:fontFamily">@font/roboto_family</item>
         <item name="android:textSize">@dimen/vtmn_text_size_subtitle_1</item>
         <item name="lineHeight">@dimen/vtmn_line_height_subtitle_1</item>
         <item name="android:textStyle">bold</item>
-        <item name="fontWeight">700</item>
         <item name="android:textColor">?attr/vtmnContentPrimary</item>
     </style>
 
     <style name="TextAppearance.Vitamin.Subtitle2" parent="@style/TextAppearance.MaterialComponents.Subtitle2">
-        <item name="fontFamily">@font/roboto_regular</item>
-        <item name="android:fontFamily">@font/roboto_regular</item>
+        <item name="fontFamily">@font/roboto_family</item>
+        <item name="android:fontFamily">@font/roboto_family</item>
         <item name="android:textSize">@dimen/vtmn_text_size_subtitle_2</item>
         <item name="lineHeight">@dimen/vtmn_line_height_subtitle_2</item>
         <item name="android:textStyle">normal</item>
-        <item name="fontWeight">400</item>
         <item name="android:textColor">?attr/vtmnContentPrimary</item>
     </style>
 
     <style name="TextAppearance.Vitamin.Body1" parent="@style/TextAppearance.MaterialComponents.Body1">
-        <item name="fontFamily">@font/roboto_regular</item>
-        <item name="android:fontFamily">@font/roboto_regular</item>
+        <item name="fontFamily">@font/roboto_family</item>
+        <item name="android:fontFamily">@font/roboto_family</item>
         <item name="android:textSize">@dimen/vtmn_text_size_body_1</item>
         <item name="lineHeight">@dimen/vtmn_line_height_body_1</item>
         <item name="android:textStyle">normal</item>
-        <item name="fontWeight">400</item>
         <item name="android:textColor">?attr/vtmnContentPrimary</item>
     </style>
 
+    <style name="TextAppearance.Vitamin.Body1.Bold">
```

Comment:
Why this style? (and same for Body2 and 3)

Reply:
Hello 😄 
For your information, with the Vitamin migration on our application, our UX designer has used some bold on Body styles and I finally ended to create those body styles in order to write one line instead of two (style + textStyle) but I do not know if it is the best practice about styles

Reply:
After discussion with Vitamin designer and @GerardPaligot, we don't want to create custom text appearance style for every variant. As `Body1Bold` is just the `Body1` with bold style, it's more intuitive in Android to keep the `Body1` and set the style as `bold`. Custom style should be used to define a completely different text appearance and not a simple font-weight change.

@Open-Deca-JulienGarnier You can keep your style if it is more simple to use to you, it's a project side choice, not Vitamin 😃 

(For information, the Body1Bold variant exists on Figma to keep the link with Vitamin -> if the design overrides the font-weight of a Vitamin text style, it breaks the link with Vitamin Figma).